acceptance/lib/puppet/acceptance/common_utils.rb

```diff
@@ -1,5 +1,160 @@
 module Puppet
   module Acceptance
+    module BeakerUtils
+      # TODO: This should be added to Beaker
+      def assert_matching_arrays(expected, actual, message = "")
+        assert_equal(expected.sort, actual.sort, message)
+      end
+
+      # TODO: Remove the wrappers to user_present
+      # and user_absent if Beaker::Host's user_present
+      # and user_absent functions are fixed to work with
+      # Arista (EOS).
+
+      def user_present(host, username)
+        case host['platform']
+        when /eos/
+          on(host, "useradd #{username}")
+        else
+          host.user_present(username)
+        end
+      end
+
+      def user_absent(host, username)
+        case host['platform']
+        when /eos/
+          on(host, "userdel #{username}", acceptable_exit_codes: [0, 1])
+        else
+          host.user_absent(username)
+        end
+      end
+    end
```

Comment:
It looks like the only net-new stuff we should be including in for this file is Enis' addition, that ends here

Reply:
@melissa so everything after this line that's an addition should be removed?